### PR TITLE
added displaced variables in jet hist

### PR DESCRIPTION
**Pull request by hamzahanif2210**
added displaced variables in jet hist

### CODE REVIEW

File: Root/JetHists.cxx [hunk 1]
```diff
@@ -55,6 +55,18 @@ StatusCode JetHists::initialize() {
...
   }
 
+
+
+  // details for displace jet information
+  if( m_infoSwitch->m_displaced ) {
+    if(m_debug) Info("JetHists::initialize()", "adding displaced plots");
+    m_ipsig      = book(m_name, "IPSIG",         m_titlePrefix+"Impact Point Significance" ,    120, 0, 5);
+    m_chf       = book(m_name, "CHF",          m_titlePrefix+"Charge Hadron Fraction" ,     120, 0, 2);
+    m_aplha_max   = book(m_name, "Alpha_max",      m_titlePrefix+"#alpha_{max}" , 120, 0, 1);
```
Comment (jburzy01) [inline Root/JetHists.cxx]: this variable name is misspelled

**Comment by jburzy01** [comment]
Hi @hamzahanif2210. You also need to add the part where you fill the histograms like you had in the previous implementation. Currently this will just create the histograms but not do anything with them.

**Comment by hamzahanif2210** [comment]
> Hi @hamzahanif2210. You also need to add the part where you fill the histograms like you had in the previous implementation. Currently this will just create the histograms but not do anything with them.

Done

**Review by jburzy01 — COMMENTED** [review]
your jet variables should not have "jet" in front of the variable name to be consistent with other member variables here

**Pull request closed without merging**